.changeset/mighty-apricots-burn.md

```diff
@@ -0,0 +1,6 @@
+---
+'@crowdstrike/glide-core': patch
+---
+
+- Multiselect Dropdown no longer shows the checkboxes of selected but disabled options as checked. This change brings what the user sees as selected in line with what's submitted with the form. Same change and thinking for single-select Dropdown.
```

Comment:
It's a divergence from native. But do you think it makes more sense from the user's perspective? If so, I'll bring the same change to Checkbox Group and Radio Group.